packages/diagrams/lib/src/align.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+import 'diagram_step.dart';
+
+class AlignDiagram extends StatelessWidget implements DiagramMetadata {
+  const AlignDiagram(this.name);
+
+  @override
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+//    List<Widget> diagramChildren;
+    const Widget logo = FlutterLogo(size: 60);
+    const Icon origin = Icon(
+      Icons.fiber_manual_record,
+      size: 20,
+    );
+    Widget heading;
+    Widget containerChild;
+    switch (name) {
+      case 'align_constant':
+        heading = const Text('Alignment.topRight');
+        containerChild = const Align(
+          alignment: Alignment.topRight,
+          child: logo,
+        );
+        break;
+      case 'align_alignment':
+        heading = const Text('Alignment Origin');
+        containerChild = Stack(
+          children: const <Widget>[
+            Align(
+              alignment: Alignment(0.2, 0.6),
+              child: logo,
+            ),
+            Align(
+              alignment: Alignment.center,
+              child: origin,
+            ),
+          ],
+        );
+
+        break;
+      case 'align_fractional_offset':
+        heading = const Text('Fractional Offset Origin');
```

Comment:
I'm also a little confused about the context of this diagram: is it supposed to show where the origin is for the fractional offset?

The location of the logo looks kind of arbitrary too.